src/vm/constrainedexecutionregion.cpp

```diff
@@ -1262,6 +1262,7 @@ void PrepareCriticalFinalizerObject(MethodTable *pMT, Module *pModule)
     }
     _ASSERTE(pParent != NULL);
 
+#ifdef FEATURE_CER
```

Comment:
Would it be better to ifdef out `CerNgenRootTable::Restore` and `MethodCallGraphPreparer::Run` instead, or in addition to this? Whenever we hit either of these two methods, we seems to be exposed to the CER bugs.